### ISSUE TITLE
I'm starting to get the feeling that the big intellectual conflict of the next decade is going to be Bayesians vs. Cyberneticists, which is to say verificationism vs. falsificationism with better mathematical tools.

### ISSUE THREAD

**Issue by jesparent** [OREL-group/Cognition-Futures#12]
https://twitter.com/deontologistics/status/1359986454013829123

**Comment by jesparent** [comment]
@Orthogonal-Research-Lab  @balicea  I think this might be  a fun thing to turn into an article somewhere, we've had a lot of good discussion about this topic, even a bit more from last meeting. Thoughts?

**Comment by jesparent** [comment]
maybe more magazine ish?

**Comment by Orthogonal-Research-Lab** [comment]
Yes, I have something in the back of mind right now. Something that also includes far-from-equilibrium systems. More soon.

**Comment by jesparent** [comment]
As December update, this is still on my mind, and I think I would like to flesh this out in an article of some refinement and development in the next month or so .
